libp2p/switch.nim

```diff
@@ -412,16 +333,16 @@ proc internalConnect(s: Switch,
     # unworthy and disconnects it
     raise newLPStreamClosedError()
 
-  await s.triggerPeerEvents(peerId, PeerEvent.Joined)
-  await s.triggerConnEvent(
+  await s.connManager.triggerPeerEvents(peerId, PeerEvent.Joined)
```

Comment:
it should probably be connManager that triggers these eventually

Reply:
yep, thats the plan